Makefile

```diff
@@ -73,6 +73,10 @@ test-node:
 test-ci-docker: launch-local-docker
 	@./scripts/run-ci-test.sh
 
+.PHONY: test-release-ci--docker ## Run CI tests with docker without clean-up
+test-release-ci-docker: launch-local-docker
+	@./scripts/run-ci-test.sh -release
```

Comment:
I'd prefer to use `release` here, hyphen might cause some parsing problems on some systems.
Later in `run-ci-test.sh` something like:
```
TYPE=
[ ! -z "$1" ] && TYPE="-$1"
```